pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "modeltestSDK"
-version = "2.1.0"
+version = "2.1.1"
```

Comment:
Strictly speaking this is `minor` upgrade not a `patch` considering that new functionality is added. 2.2.0?

Reply:
Agreed